docs/tutorials/services/plan-motion-with-arm-gripper.md

```diff
@@ -209,10 +209,6 @@ await motion_service.move(component_name=my_arm_resource, destination=test_start
 
 {{% /tab %}}
 {{% tab name="Go" %}}
-Because of ongoing experimental API changes, we must pass in a `slam` service resource name when using the Motion service to `Move`.
-Add `"go.viam.com/rdk/services/slam"` to your import list to provide access to the [SLAM Service](/services/slam).
-This tutorial will not cover any other SLAM content.
-
 ```go {class="line-numbers linkable-line-numbers"}
```

Comment:
```suggestion

```go {class="line-numbers linkable-line-numbers"}
```
for the linter :)

Reply:
Got it in another commit at the same time :+1: